book/website/collaboration/research-infrastructure-roles.md

```diff
@@ -1,4 +1,14 @@
 (cl-research-infrastructure-roles)=
+
+```{figure} ../../figures/researchers-environment*
```

Comment:
```suggestion
```{figure} ../../figures/researchers-environment.*
```

Reply:
Thank you!